src/components/containers/visibility.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ReactElement, useEffect, useState } from 'react'
+import styled from 'styled-components'
+import device, { size } from 'themes/device'
+import { useBrowserResize } from 'components/hooks/use-browser-resize'
+
+type ResponsiveContainerProps = {
+    children: ReactElement
+}
+
+const DesktopLayer = styled.div`
+    @media ${device.tablet} {
+        display: none;
+    }
+`
+const MobileLayer = styled.div`
+    @media ${device.mobile} {
+        display: none;
+    }
+`
+
+const deviceRenderer = (): boolean => {
+    const [is_loaded, setIsLoaded] = useState<boolean>(false)
```

Comment:
```suggestion
    const [is_loaded, setLoaded] = useState(false)
```
- AFAIK, the convention is that we usually remove the `is` & `has` for the setter.
- We can let TS infer the type here.